### PR TITLE
Adding webpack code splits example

### PR DESCRIPTION
**Pull request by internettrans**
image

### CODE REVIEW

File: systemjs/webpack-code-splits/src/set-public-path.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { setPublicPath } from 'systemjs-webpack-interop'
```
Comment (internettrans) [inline systemjs/webpack-code-splits/src/set-public-path.js]: I'm open to changing this not to use systemjs-webpack-interop to show the raw usage.

**Comment by internettrans** [comment]
@guybedford this is ready for your review

File: systemjs/webpack-code-splits/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+module.exports = systemjsInterop.modifyWebpackConfig({
+  entry: path.resolve(__dirname, 'src/entry.js'),
+  mode: 'development',
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  devtool: 'sourcemap',
+  plugins: [new CleanWebpackPlugin()],
```
Comment (Flamingo09) [inline systemjs/webpack-code-splits/webpack.config.js]: Where is the code to split the bundle?

Comment (internettrans) [reply systemjs/webpack-code-splits/webpack.config.js]: Here: https://github.com/systemjs/systemjs-examples/blob/bf70097688e4c062683d31c9f58399aee6747a49/optimized-builds/webpack-code-splits/src/entry.js#L12

**Pull request merged**